x-pack/plugins/alerting/README.md

```diff
@@ -145,6 +147,17 @@ server.newPlatform.setup.plugins.alerting.registerType({
 			threshold: schema.number({ min: 0, max: 1 }),
 		}),
 	},
+	actionGroups: [
```

Comment:
The formatting in GitHub for this is a bit off. Could be a mix of spaces and tabs?